### PR TITLE
Fix AWS pool configuration

### LINKED ISSUE #379: AWS K8s install docs are wrong
http://docs.projectcalico.org/v2.0/getting-started/kubernetes/installation/aws

says to run:
`docker run --rm --net=host calico/ctl pool add 192.168.0.0/16 --nat-outgoing`

That command doesn't work in the Go-version of calicoctl.

### PR DESCRIPTION
**Pull request by caseydavenport**
Fixes #379

### CODE REVIEW

File: v2.0/getting-started/kubernetes/installation/aws.md [hunk 1]
```diff
@@ -278,23 +278,32 @@ Verify that you can access the Kubernetes API.  The following command should ret
 ```shell
 ./kubectl get nodes
 ```
-### 4.2 Configure Outbound NAT
+
+## 5. Install Addons
+
+{% include {{page.version}}/install-k8s-addons.md %}
```
Comment (djosborne) [inline v2.0/getting-started/kubernetes/installation/aws.md]: install-k8s-addons ends with "Next Steps" which send you off to the k8s network policy guide. Doesn't make sense to then have \#6 after it

Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/installation/aws.md]: yuck, you're right.

File: v2.0/getting-started/kubernetes/installation/aws.md [hunk 2]
```diff
@@ -278,23 +278,32 @@ Verify that you can access the Kubernetes API.  The following command should ret
...
+
+## 6. Configure the Calico IP pool
```
Comment (djosborne) [inline v2.0/getting-started/kubernetes/installation/aws.md]: We've already disabled Source/Dest checks. Why not point out this is only required if going Cross VPC, and link to the new AWS guide for an explanation?

File: v2.0/getting-started/kubernetes/installation/aws.md [hunk 3]
```diff
@@ -278,23 +278,32 @@ Verify that you can access the Kubernetes API.  The following command should ret
...
 
-```shell
+```
 # Log into the master instance.
 ssh -i ~/mykey.pem core@$MASTER_DNS
 
-# Enable outgoing NAT on the Calico pool.
-docker run --rm --net=host calico/ctl pool add 192.168.0.0/16 --nat-outgoing
+# Enable outgoing NAT and ipip on the Calico pool.
+docker run --rm --net=host calico/ctl:v1.0.0-rc2 apply -f -<<EOF
```
Comment (robbrockbank) [inline v2.0/getting-started/kubernetes/installation/aws.md]: You also need a -i flag for this to work

```
$ docker run -i --rm --net=host calico/ctl:latest apply -f - <<EOF
apiVersion: v1
kind: ipPool
metadata:
  cidr: 192.168.0.0/16
spec:
  ipip:
    enabled: true
  nat-outgoing: true
EOF

Successfully applied 1 'ipPool' resource(s)
```

File: v2.0/getting-started/kubernetes/installation/aws.md (hunk 2 above)
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/installation/aws.md]: I think we just want to remove ipip.  This guide only does single VPC subnet anyway.

**Comment by caseydavenport** [comment]
@djosborne @robbrockbank ready for another look please!

**Comment by djosborne** [comment]
Nice, LGTM

**Pull request merged**